lib/internal/code_integrity.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
A comment here explaining a bit about what this is would be helpful. Also, if this feature is limited to windows, comments indicating so should be added as well as possible asserts to ensure it is only running on windows.

Reply:
I think this file should throw if required by a non-windows environment.